pipeline/notification/alert.py

```diff
@@ -0,0 +1,71 @@
+'''A script that sends out SNS alerts when new games come in.'''
+
+from os import environ as ENV
+
+from dotenv import load_dotenv
+from boto3 import client
+
+TAG_ARNS = {'Action': 'arn:aws:sns:eu-west-2:129033205317:c10-games-action-tag',
+            'Adventure': 'arn:aws:sns:eu-west-2:129033205317:c10-games-adventure-tag',
+            'Indie': 'arn:aws:sns:eu-west-2:129033205317:c10-games-indie-tag',
+            'Horror': 'arn:aws:sns:eu-west-2:129033205317:c10-games-horror-tag'}
+
+
+def send_sns(topic: str, games: list, config):
+    '''Sends a SNS message to subscribers of a given topic
+    about newly released games.'''
+
+    sns = client(
+        "sns", aws_access_key_id=config["AWS_KEY"], aws_secret_access_key=config["AWS_SECRET"])
+    sns.publish(
+        TargetArn=TAG_ARNS[topic],
+        Message=f'''Hey there!
```

Comment:
To clean this up, you could use `\n` instead of actual newlines?

Reply:
I did try to! But the publish functionality doesn't like backslash characters unfortunately. I will see if there is another way around this.